debug.Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM composer@sha256:d374b2e1f715621e9d9929575d6b35b11cf4a6dc237d4a08f2e6d1611f534675 as composer
```

Comment:
In general not a super big fan of having two docker images we need to keep in sync, I think this could be solved by reusing the existing docker file but adding the required dependencies and then enabling them by some volume / --build-argument thing possibly?

Reply:
I've done something similar to what you want to do (i think) in the release pipeline where a --build-argument is used as a conditional to either install dependencies or not.

https://github.com/wmde/wikibase-release-pipeline/blob/main/Docker/test/selenium/Dockerfile#L13

Reply:
This was then triggered on build by adding it here https://github.com/wmde/wikibase-release-pipeline/blob/main/test/run_selenium.sh#L40 Maybe something like that could work?

Reply:
The alternative is to build the debug image from the production image (e.g. `FROM ghcr.io/wbstack/api`) then you don't need to copy pasta everything over and keep in sync. Not sure if which method I prefer though.